Solar_radiation_scripts/retrieve_DEOS_data_v2.R

```diff
@@ -2,72 +2,102 @@ library(XML)
 library(dplyr)
 library(lubridate)
 
-# get a sequence of dates
-# looks like you can only get solar radiation from hourly data
-
-all_days <- seq(as.Date("2015-01-01"), as.Date("2015-12-31"), by="days")
-
-base_url <- 'http://deos.udel.edu/odd-divas/station_daily.php?network=DEOS&station=DWCH&'
-dat_all <- data.frame()
-
-for (i in 1:length(all_days)) {
-  date_extension <- sprintf('month=%s&day=%s&year=%s', lubridate::month(all_days[i]), lubridate::day(all_days[i]), lubridate::year(all_days[i]))
-  new_url <- paste0(base_url, date_extension)
-
-  # export message so you know what's going on
-  message('Retrieving data for ', all_days[i])
-
-  # get that day's data
-  dat_list <- XML::readHTMLTable(new_url, stringsAsFactors = FALSE)
-  dat <- dat_list[[2]][c(1,3,6,17,19)] %>%
-    mutate(date = all_days[i])
-
-  dat_all <- rbind(dat, dat_all)
+# check wd
+getwd()
+
+# Set wd
+setwd('..')
```

Comment:
Hi Jake - sorry for messing around with the working directory. I talked this over with Sam, but basically I was checking the wd here because in another script, where I am pulling PSM data, the function automatically drops data files in your root directory, so I was having to reset the wd in that script, which was then messing me up in this script. BUT Sam pointed out that I can change and then reset the directory within my retrieval function in the other script, which should clear up the issue. So these lines can be removed. I'll update that.

Reply:
Ah gotcha, that makes sense. Thanks for explaining!